src/containers/LoginRedirectPage.tsx

```diff
@@ -47,19 +47,21 @@ export class LoginRedirectPage extends React.Component<LoginRedirectPageProps, L
     const locationData = window.location.search;
     const url = locationData;
     const params = queryString.parse(url);
-    if (Config.b2b.openId && Config.b2b.openId.enable && params.code && params.session_state) {
+    const b2bEnabled = Config.b2b.openId && Config.b2b.openId.enable;
```

Comment:
This is checking if openID is enabled, not if b2b is enabled. We have a separate config for that
```suggestion
    const openidEnabled = Config.b2b.openId && Config.b2b.openId.enable;
```